### PR TITLE
1) Stop fix; 2) if txid and ind are missing - asking to generate new address for collateral tnx and waiting for it 3) asking for passphrase if missing

### PR DESCRIPTION
**Pull request by a-ok123**

### CODE REVIEW

**Review by mateeullahmalik — APPROVED** [review]

File: cmd/common.go [hunk 1]
```diff
@@ -322,3 +325,97 @@ func GetPastelInfo(ctx context.Context, config *configs.Config) (structure.RPCGe
...
+// CheckPastelDRunning whether pasteld is running
+func CheckPastelDRunning(ctx context.Context, config *configs.Config) (ret bool) {
+	var failCnt = 0
+	var err error
+
+	log.WithContext(ctx).Info("Waiting the pasteld to be started...")
+
+	for {
+		if _, err = RunPastelCLI(ctx, config, "getinfo"); err != nil {
+			time.Sleep(10000 * time.Millisecond)
```
Comment (nguyen202) [inline cmd/common.go:338]: could you simply change  `10000 * time.Millisecond` to `10*time.Second`?

File: cmd/start.go [hunk 2]
```diff
@@ -752,53 +863,66 @@ func createOrUpdateMasternodeConf(ctx context.Context, config *configs.Config) (
...
+		log.WithContext(ctx).WithError(err).Error("Failed to create new masternode.conf file")
 		return err
 	}
 	defer confFile.Close()
 
+	_, err = confFile.Write(confData)
+	if err != nil {
+		log.WithContext(ctx).WithError(err).Error("Failed to write data in the new masternode.conf file")
+		return err
+	}
```
Comment (nguyen202) [inline cmd/start.go:917]: nit: replace os.Create + .Write by ioutil.WriteFile()

**Review by nguyen202 — DISMISSED** [review]
LGTM!

File: cmd/common.go (hunk 1 above)
Comment (a-ok123) [reply cmd/common.go:338]: ok, will change - that was in code that way already :)

**Review by mateeullahmalik — APPROVED** [review]

**Pull request merged**